github_tutorial.md

```diff
@@ -173,6 +192,21 @@ De beheerder van de standaard zal dus de volgende handelingen doen voor een vers
 		* Beschrijf hoe mensen aan de ontwikkeling kunnen bijdragen (contributing guide).
 	* Maak, zover mogelijk een goed begin. Mensen dragen makkelijker bij aan iets dat er al (in enig vorm) is, dan aan een lege repository.
 	* Nodig relevante belanghebbenden (potentiële gebruikers of deelnemers) uit om het "product" in de repository te gebruiken en eraan bij te dragen.
+	
+19. Ik wil voorafgaand aan een nieuwe actie mijn fork updaten.
+	* Open de fork die je wil updaten;
+	* Klik op de tab "Pull Requests";
+	* Klik op "New Pull Request". GitHub vergelijkt nu de originele repository met je fork.
+	  Als het goed is zijn er geen verschillen;
+	* Klik op "switching the base". GitHub vergelijkt nu je fork met de originele repository.
+	  Als er tussentijds wijzigingen zijn aangebracht dan zou je nu wel verschillen moeten zien.
+	* Klik op "Create pull request" en ken er een duidelijke naam aan toe;
+	* Scroll evt. naar beneden en klik op "Create pull request";
+	* Scroll naar beneden naar "Merge pull request";
+	* Kies een van de 3 opties
```

Comment:
Als je doel is de fork te synchroniseren met de VNG-R repository heeft echt alleen de eerste optie zin. De rest is voor problemen en uitzonderingssituaties waar men hier niks aan heeft (want iemand die de tutorial leest weet nog niet wanneer en waarvoor deze 2e of 3e optie gebruikt moet worden). Ik zou zeggen dat ze gewoon op "Merge pull request" moeten klikken.

Reply:
Is aangepast.